### PR TITLE
Make a videojs preset based on angular preset

### PR DESCRIPTION
**Pull request by gkatsev**
The main difference is that it allows all of the known commit groups into the changelog. It then sorts them by BREAKING CHANGE, then Features, then Bug Fixes, finally following by each other commit group sorted by the group's title.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -103,7 +103,36 @@ var writerOpts = {
     return commit;
   },
   groupBy: 'type',
-  commitGroupsSort: 'title',
+  commitGroupsSort: function(group1, group2) {
```
Comment (gkatsev) [inline index.js:106]: Can't think of an easier way to bubble up things in order of:
Breaking, Features, Bug, {{title}}

File: index.js [hunk 2]
```diff
@@ -103,7 +103,36 @@ var writerOpts = {
...
+    }
+
+    if (group1.title === 'Features') {
+      return -1;
+    }
+    if (group2.title === 'Features') {
+      return 1;
+    }
+
+    if (group1.title === 'Bux Fixes') {
```
Comment (misteroneill) [inline index.js]: Typo!

File: index.js (hunk 1 above)
Comment (misteroneill) [reply index.js:106]: What about:

``` js
const titles = [
  'BREAKING CHANGES',
  'Features',
  'Bug Fixes'
];

for (let i = 0; i < titles.length; i++) {
  if (group1.title === titles[i]) {
    return -1;
  }
  if (group2.title === titles[i]) {
    return 1;
  }
}
```

**Review by misteroneill — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (gkatsev) [reply index.js:106]: Yeah, looks like that would simplify it a bit.

File: index.js (hunk 2 above)
Comment (gkatsev) [reply index.js]: oops.

File: index.js (hunk 1 above)
Comment (imbcmdth) [reply index.js:106]:
```
const titles = [
  'BREAKING CHANGES',
  'Features',
  'Bug Fixes'
];

return Math.sqrt(titles.indexOf(group1.title)) - Math.sqrt(titles.indexOf(group2.title));
```

Comment (imbcmdth) [reply index.js:106]: Wait that's not it... so close!

Comment (misteroneill) [reply index.js:106]: You're a madman @imbcmdth!

Comment (gkatsev) [reply index.js:106]: I went with the loop approach.

**Pull request merged**